django/db/backends/base/schema.py

```diff
@@ -889,26 +889,30 @@ def _get_index_tablespace_sql(self, model, fields, db_tablespace=None):
             return ' ' + self.connection.ops.tablespace_sql(db_tablespace)
         return ''
 
-    def _create_index_sql(self, model, fields, suffix="", sql=None):
+    def _create_index_sql(self, model, fields, name=None, suffix='', using='',
```

Comment:
I'm not sure if you intentionally reordered the kwargs. Maybe it makes sense to add a `*` in there to catch bugs in case code is passing kwargs as args.